src/TesApi.Web/tes-compliance-test/entrypoint.sh

```diff
@@ -3,7 +3,7 @@ teshostname=$(jq -r '.TesHostname' TesCredentials.json)
 tesuser=$(jq -r '.TesUsername' TesCredentials.json)
 tespassword=$(jq -r '.TesPassword' TesCredentials.json)
 
-# This file will be used to run tes-compliance-tests as part of the ADO release
-# TES-compliance-suite repo provides a default entry point script, but since TES uses basic auth it needs to be replaced to add approprite credentials
-# https://github.com/elixir-cloud-aai/tes-compliance-suite
-tes-compliance-suite report --server http://$tesuser:$tespassword@$teshostname/ --tag all --version 1.0.0 --output_path results
\ No newline at end of file
+# This file will be used to run tes compliance tests as part of the ADO release
+# openapi-test-runner repo provides a default entry point script, but since TES uses basic auth it needs to be replaced to add approprite credentials
+# https://github.com/elixir-cloud-aai/openapi-test-runner
+openapi-test-runner report --server http://$tesuser:$tespassword@$teshostname/ --tag all --version 1.0.0 --output_path results
```

Comment:
i asume we are still on version tes 1.0.0, is that correct?

Reply:
Great point, yes, and I wonder if we could loop through each TES version (1.0.0, 1.1.0), run the `openapi-test-runner`, and update the status of a badge on the readme that says "✔️ GA4GH TES 1.0.0", "✔️ GA4GH TES 1.1.0"

Reply:
we certainly can - but this is a bit more of a lift. Great opportunity for someone to pick this up and learn how these tests work (so I am not the only one with this knowledge). I'll make an issue to track this and happy to help if any road blocks are hit.